.travis.yml

```diff
@@ -6,9 +6,9 @@ python:
   - "3.5"
 # command to install dependencies
 install:
-  - cd tests
   - sudo apt-get -y install python3-pip
-  - pip3 install bs4
+  - pip3 install -r requirements.txt
+  - cd tests
```

Comment:
Since we are using pytest, whenever you go into the test suite. You only need to run `pytest` to run all of the tests.

Reply:
Fixed